File: samples/debugTest0.cpp
#include <iostream>
using namespace std;

int main(void)
{
    int i;
    float result = 0.0;
    for (i = 1; i < 10; i++)
    {
        float tmp = 1 / i;
        result = result + tmp;
    }
    cout << "result = " << result << endl;
    return 0;
}

File: debug/DebugTest0.java
package samples;

public class DebugTest0 {
    public static void main(String[] args) {
        double result = 0.0;
        for (int i = 1; i < 10; i++) {
            float tmp = 1 / i;
            result = result + tmp;
        }
        System.out.println("result = " + result);
    }
}
